internal/impl/kafka/enterprise/redpanda_migrator_output.go

```diff
@@ -471,12 +485,12 @@ func (w *RedpandaMigratorWriter) WriteBatch(ctx context.Context, b service.Messa
 			return
 		}
 		if w.key != nil {
```

Comment:
Is it better to change this check to look at `keyExecutor != nil`? Same in other places.

Reply:
Thank you! Yeah... Guess this doesn't matter much since it will get dropped once #2918 is merged.